packages/web-components/fast-components/src/anchored-region/anchored-region.definition.ts

```diff
@@ -17,21 +17,27 @@ export const fastAnchoredRegionDefinition: WebComponentDefinition = {
             attributes: [
                 {
                     name: "anchor",
+                    title: "Anchor",
```

Comment:
Out of scope for this review, but this is an unfortunate name given the `fast-anchor` component. I think _that_ component actually has the wrong name, which obviously goes back to the the early days of the html spec. That thing should be a link or something instead as it's rarely used to anchor to a position on the page.

I can't think of a great synonym here though.

Reply:
Good point, I made a suggestion that might help clarify.